WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -253,7 +247,7 @@ public void Start(TimeSpan requestInterval, TimeSpan feeQueryRequestInterval, in
 
 							hashChain.UpdateServerTipHeight((uint)response.BestHeight);
 							ExchangeRate? exchangeRate = response.ExchangeRates.FirstOrDefault();
-							if (exchangeRate is { } && exchangeRate.Rate != 0)
+							if (exchangeRate is { Rate: > 0 })
```

Comment:
I think this is okay as it is but I just learnt that it is also possible to us `if (exchangeRate is { Rate: not 0 })`